BinaryEncoding.md

```diff
@@ -81,7 +81,7 @@ Yes:
   - A table (sorted by offset) containing, for each type which has opcodes:
     + A standardized string literal [type name](AstSemantics.md#local-and-memory-types).
       The index of a type name in this table is referred to as a type ID
-    + 32-bit offset of its opcode table within the section
+    + 64-bit offset of its opcode table within the section
```

Comment:
Fixed-size fields in structures are arguably a virtue compared to everything being variable-length, though. Decoding is simpler and you can seek to known offsets. In a scenario like this saving a couple bytes by using a variable-length integer instead of just a 64-bit one would be... hm.